solr/solr-ref-guide/modules/deployment-guide/pages/metrics-reporting.adoc

```diff
@@ -879,7 +879,7 @@ The response will look like this:
   ]}
 ----
 
-Like other request handlers, the Metrics API can also take the `wt` parameter to define the output format.
+Like other request handlers, the Metrics API can also take the `wt` parameter to define the output format. The Metrics API also provides an additional special `wt` parameter `prometheus` that will output metrics in Prometheus format. This can be used for xref:monitoring-with-prometheus-and-grafana.adoc#metrics-api-with-prometheus-format[Monitoring with Prometheus and Grafana]
```

Comment:
After each sentence, put a newline (doesn't change the final render but it to keep editing & diffs more manageable).

Should document what options *do not work* with prometheus.  The regex one doesn't, you said.

Reply:
Added in a small note about the filter query parameters. Let me know if you prefer a different way to write it on the ref-guide